### PR TITLE
[Try] BERT fine-tuning model

### PR DESCRIPTION
**Pull request by adamziel**
Just an early draft.

### CODE REVIEW

**Comment by adamziel** [comment]
This is quite memory-heavy:

Zrzut ekranu 2020-11-25 o 00 22 23

It's most likely related to all the stored gradients, but I would like to double-check because of the memory problems we experienced in the embeddings model. I checked a few other PyTorch implementations but none of them is maintained and easy/appropriate to reuse. There is an official TensorFlow implementation though that I may try next:

https://www.tensorflow.org/official_models/fine_tuning_bert

**Comment by ItsWayneYang** [comment]
I tried running with batch size 32 and got this error
Screen Shot 2020-11-25 at 1 11 49 PM

Did you see this error before?
